<!-- -->
<launch>
  <arg name="sensor_frame" default="/base_link" />
  <arg name="input1" default="/front/velodyne_points" />
  <arg name="input2" default="/rear/velodyne_points" />
  <arg name="output" default="/points_raw" />

  <node pkg="points_preprocessor" type="fusion_filter" name="fusion_filter" output="log">
    <param name="sensor_frame" value="$(arg sensor_frame)" />
    <remap from="/front/velodyne_points" to="$(arg input1)" />
    <remap from="/rear/velodyne_points" to="$(arg input2)" />
    <remap from="/points_raw" to="$(arg output)" />
  </node>
</launch>
